backend/src/main/kotlin/cz/loono/backend/api/service/AccountService.kt

```diff
@@ -158,4 +158,22 @@ class AccountService(
             page = accountsPage.nextPageable()
         } while (accountsPage.hasNext())
     }
+
+    fun login(uid: String) {
+        val account = accountRepository.findByUid(uid) ?: throw LoonoBackendException(
+            status = HttpStatus.NOT_FOUND,
+            errorCode = "404",
+            errorMessage = "The account not found."
+        )
+        accountRepository.save(account.copy(active = true))
```

Comment:
What about using an expression?
```suggestion
    fun login(uid: String) =
        accountRepository.findByUid(uid)?.let {
            accountRepository.save(it.copy(active = true))
        } ?: throw404() 
        
    private fun throw404(): Nothing {
        throw LoonoBackendException(
            status = HttpStatus.NOT_FOUND,
            errorCode = "404",
            errorMessage = "The account not found."
        )
    }
```